app/views/doc/ext.html.erb

```diff
@@ -9,12 +9,6 @@
     <div class='column-left'>
       <h3>Short &amp; Sweet</h3>
       <ul class='content-list'>
-        <li>
-          <h4><a href="http://gitref.org/">Introductory Reference &amp; Tutorial</a></h4>
```

Comment:
why don't we also provide the web archive link? https://web.archive.org/web/20161016050919/http://gitref.org:80/

on the other hand, if we keep like this, we may get to a point where many links are of web archive. I would say that for now it works, but some day we will have simply to replace with new tutorials

Reply:
I didn't do that as the site "redirected" and was not dead.

>on the other hand, if we keep like this, we may get to a point where many links are of web archive. I would say that for now it works, but some day we will have simply to replace with new tutorials

Yeah, I did anticipate that. So, may be we should start adding archived links and start removing the broken links from a point of time. That makes me ask, why not now? So, how about removing the "Git for Designers" item?

Reply:
I was curious what happened with gitref, so I dug into this a little bit. Looks like it has been largely unmaintained, and the domain was even down for quite a while. So the GitHub folks who own it just took it down and redirected the domain.

The content is still available at http://git.github.io/git-reference/, so we _could_ link there. But as nobody is really maintaining it, I'd just as soon avoid it and steer people towards the manpages or the book content already on the site.